### ISSUE TITLE
Zero Unit rule false positive on zeros

### ISSUE THREAD

**Issue by bgriffith** [sasstools/sass-lint#75]
When `include` is set to `true` the following `0` is incorrectly reported as requiring a unit:
```scss
// False positive
@if $foo == 0 {
  content: '...';
}
```
